### ISSUE TITLE
Fail2ban: broken by missing nethserver-cockpit library

### ISSUE THREAD

**Issue by stephdl** [NethServer/dev#5813]
**Steps to reproduce**

- install nethserver-cockpit-0.11.0-1.54.g804ec75.ns7.noarch from testing
- install nethserver-fail2ban
- yum complains about a missing library with a perl require

```
Error: Package: nethserver-fail2ban-1.1.7-1.ns7.noarch (nethserver-updates)
           Requires: /usr/libexec/nethserver/api/lib/auth_functions.pl
           Available: nethserver-cockpit-0.1.1-1.ns7.noarch (nethserver-base)
               Not found
```

**Expected behavior**

I expect no errors

**Actual behavior**

Due to a work on [nethserver-cockpit and delegation of authority](https://github.com/NethServer/nethserver-cockpit/pull/117/commits/1244559613b16db14c2b6b732fb1bf3e993b0224) the library has been removed, but the rpm has been pushed to testing

To fix it

- Either remove the rpm from testing (nethserver-cockpit-0.11.0-1.54.g804ec75) and wait that the PR https://github.com/NethServer/nethserver-cockpit/pull/117 has been approved and merged (the fix is already done but not merged in https://github.com/NethServer/nethserver-fail2ban/pull/65)
- Or commit, merge and build a rpm to nethserver-fail2ban to remove[ the require to that library](https://github.com/NethServer/nethserver-fail2ban/pull/65/commits/012c309f24f0fa0dc353e020d6cb0f0d3bd01392)

**Components**

nethserver-cockpit-0.11.0-1.54.g804ec75
nethserver-fail2ban-1.1.7-1.ns7.noarch 

----

thank @filippocarletti

**Comment by stephdl** [comment]
QA

@gsanchietti has removed the rpm nethserver-cockpit from testing and buildt another one from master. I now can install nethserver-cockpit and fail2ban

```
 [root@ns7loc14 ~]# rpm -qa nethserver-fail2ban nethserver-cockpit nethserver-cockpit-lib
nethserver-cockpit-lib-0.11.0-1.11.g897dba9.ns7.noarch
nethserver-fail2ban-1.1.7-1.ns7.noarch
nethserver-cockpit-0.11.0-1.11.g897dba9.ns7.noarch
```

set closed
